Lib/MultiCounter.hpp

```diff
@@ -56,6 +56,7 @@ class MultiCounter
     if (v >= _top) { // not enough capacity
       expandToFit(v);
     }
```

Comment:
Not actually required, obviously. In some convoluted cases the static analyser gets confused - I think this is worth the extra lines, it's not much cost in debug and zero cost in release.